OCR/florence/utils.py

```diff
@@ -0,0 +1,39 @@
+
+UID_KEY = "UID"
+MESSAGE_KEY = "Message"
+CONTAINS_KEYWORDS_KEY = "Contains Keywords"
+EXTRACTED_DATA_KEY = "Extracted Data"
+
+
+def add_words_counts(words_counts,response_words_counts):
+
+    for key,value in response_words_counts:
+        words_counts[key] += value
```

Comment:
Hi, 
I don't think we need to add the values in the key here. 
each key just tells if a word is present or not. basically if sum of values > 0, then it should be 1 and if sum (numbers) = 0, then 0